rule.go

```diff
@@ -42,6 +42,7 @@ type Rule struct {
 	Chain    *Chain
 	Position uint64
 	Handle   uint64
+	Flags    uint32
```

Comment:
Can you add a comment to this field describing which flags are valid to specificy here or which ones the caller should expect? AFAICT, thus far only the NFTA_RULE_POSITION flag plays any role, right?